### PR TITLE
Fix layout

### PR DESCRIPTION
**Pull request by kazup01**
I fixed following layout.

- InfoTab
- Export icon at InfoPanel
- Overall button

### CODE REVIEW

File: browser/main/modals/PreferencesModal/InfoTab.js [hunk 1]
```diff
@@ -96,16 +96,16 @@ class InfoTab extends React.Component {
         </ul>
         <hr />
         <div styleName='policy'>Data collection policy</div>
-        <p>We collect only the number of users on Boostnote for DAU and any detail information<br />
-        such as a note's content or title is not collected.</p>
-        <p>You can see how it works on <a href='https://github.com/BoostIO/Boostnote'>GitHub</a></p>
+        <p>We collect only the number of DAU for Boostnote, NOT COLLECTED any detail information</p>
```
Comment (asmsuechan) [inline browser/main/modals/PreferencesModal/InfoTab.js]: I assume `and DO NOT COLLECT` is better.

File: browser/main/modals/PreferencesModal/InfoTab.js [hunk 2]
```diff
@@ -96,16 +96,16 @@ class InfoTab extends React.Component {
...
+        <p>like your note content. You can see how it works on <a href='https://github.com/BoostIO/Boostnote' onClick={(e) => this.handleLinkClick(e)}>GitHub</a>.</p>
```
Comment (asmsuechan) [inline browser/main/modals/PreferencesModal/InfoTab.js]: Why did you change `such as` to `like`?

File: browser/main/modals/PreferencesModal/InfoTab.js (hunk 1 above)
Comment (kazup01) [inline browser/main/modals/PreferencesModal/InfoTab.js]: Done👍

File: browser/main/modals/PreferencesModal/InfoTab.js (hunk 2 above)
Comment (kazup01) [inline browser/main/modals/PreferencesModal/InfoTab.js]: No mean. I turned back it to `such as`.

File: browser/main/modals/PreferencesModal/InfoTab.js (hunk 1 above)
Comment (asmsuechan) [inline browser/main/modals/PreferencesModal/InfoTab.js]: Excuse me again, the comma is not needed.

File: browser/main/modals/PreferencesModal/InfoTab.js (hunk 2 above)
Comment (asmsuechan) [inline browser/main/modals/PreferencesModal/InfoTab.js]: No problem.

**Comment by kazup01** [comment]
New UI
screen shot 0029-07-24 at 7 17 42 pm

File: browser/main/modals/PreferencesModal/InfoTab.js (hunk 1 above)
Comment (asmsuechan) [inline browser/main/modals/PreferencesModal/InfoTab.js]: Oh, I noticed that it's `COLLECTED`...

**Comment by asmsuechan** [comment]
Umm, it does not work fine.
image
